.changeset/empty-papayas-raise.md

```diff
@@ -0,0 +1,6 @@
+---
+"@directus/api": patch
+"tests-blackbox": patch
+---
+
+Added deletion of `dist` folder to the `build` command
```

Comment:
Required tests were somehow not triggered...

![image](https://github.com/user-attachments/assets/d376164f-38d3-485b-8598-5ad06eb1883b)

```suggestion
Added deletion of the `dist` folder to the `build` command
```